server/routes/__tests__/updates.spec.js

```diff
@@ -0,0 +1,251 @@
+const request = require('supertest');
+const cheerio = require('cheerio');
+
+jest.mock('@sentry/node');
+
+const { createUpdatesContentRouter } = require('../updates');
+const { setupBasicApp } = require('../../../test/test-helpers');
+
+describe('GET /updates', () => {
+  let app;
+
+  const cmsService = {
+    getUpdatesContent: jest.fn(),
+  };
+
+  const sessionMiddleware = (req, res, next) => {
+    req.session = {
+      establishmentId: 123,
+      establishmentName: 'berwyn',
+    };
+    next();
+  };
+
+  let relatedContent;
+  let data;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    app = setupBasicApp();
+    app.use(sessionMiddleware);
+    app.use('/', createUpdatesContentRouter({ cmsService }));
+
+    relatedContent = [
+      {
+        id: 15826,
+        contentType: 'video',
+        externalContent: false,
+        title: 'BBC. The Story of Maths. The Language of the Universe',
+        summary: 'BBC. The Story of Maths. The Language of the Universe',
+        contentUrl: '/content/15826',
+        displayUrl: undefined,
+        image: { url: 'image url', alt: 'Alt text' },
+      },
+      {
+        id: 15825,
+        contentType: 'video',
+        externalContent: false,
+        title: 'The Queen: 70 Glorious Years - British Royal Documentary',
+        summary: 'The Queen: 70 Glorious Years - British Royal Documentary',
+        contentUrl: '/content/15825',
+        displayUrl: undefined,
+        image: { url: 'image url', alt: 'Alt text' },
+      },
+    ];
+
+    data = {
```

Comment:
data doesn't appear to be modified. This should be a const in describe and not within the beforeEach?